### PR TITLE
Add the insurance company name to the demographics report

### PR DESCRIPTION
**Pull request by teryhill**
Add the insurance company name to the demographics report #778

### CODE REVIEW

**Comment by teryhill** [comment]
Sorry @aethelwulffe , @Trodrige and @tmccormi  the reports.inc file has a lot of just formatting changes. The only lines that matter is 112-140 and 473-479

**Comment by teryhill** [comment]
There is more than that one. I will fix it as a security fix for the program or maybe give it to one of the GCI folks.

File: library/report.inc [hunk 1]
```diff
@@ -109,23 +109,52 @@ subscriber_employer_state => xl('Subscriber Employer State') . ": ",
...
-			if ($ret[$key]['content'] != $value && $ret[$key]['date'] < $list['date'])
-			{
-				$ret[$key]['title'] = $key;
-				$ret[$key]['content'] = $value;
-				$ret[$key]['date'] = $list['date'];
-			}
-		}
-	}
-	return $ret;
+    $sql = "select * from patient_data where pid='$pid' order by date ASC";
```
Comment (aethelwulffe) [inline library/report.inc]: `$sql = "select * from patient_data where pid='$pid' order by date ASC";`
Change to `$sql = "select * from patient_data where pid = ? order by date ASC;";`
If that doesn't work then...
 `$sql = "select * from patient_data where pid= ".$pid." order by date ASC;";`

Comment (aethelwulffe) [reply library/report.inc]: Ooops.  That assumes a numeric pid...
Make that either the ordinal ? or use '".$pid."' (single, double,dot,var,dot, double, single).

Comment (teryhill) [reply library/report.inc]: That is outside of the scope of this pr. It you look down the file there are a huge number that need to be fixed. I offered to change them in another pr. Tony wants the pr's to be what they are, nothing added or taken away.

Comment (aethelwulffe) [reply library/report.inc]: Otay.

**Comment by teryhill** [comment]
Do y'all want this or should I delete it?

**Comment by teryhill** [comment]
@tmccormi is this wanted?

**Comment by tmccormi** [comment]
Seems fine to me.  I'll merge it in.

**Pull request merged**